integrated-tests/src/test/scala/commercial/AdsTest.scala

```diff
@@ -1,15 +1,19 @@
-package commercial
+package integration
 
-import driver.Driver
+import Config.baseUrl
 import org.scalatest.tags.Retryable
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{DoNotDiscover, FlatSpec, Matchers}
 
-
-@Retryable class AdsTest extends FlatSpec with Matchers with Driver {
+@DoNotDiscover @Retryable class AdsTest extends FlatSpec with Matchers with SharedWebDriver {
 
   "Ads" should "display on the network front" in {
 
-    go to theguardianWithAds("/uk")
+    webDriver.get(theguardianWithAds("/uk"))
+
+    // This is an essential sleep, because the implicitlyWait isn't sufficient to ensure that
+    // the js application has completed, since the dfp-ad classes exist on page load.
+    Thread.sleep(10000)
```

Comment:
Maybe, let's see if devs really need to test that kind of stuff in here. I removed the parallel factor for these tests, they don't need to be fast really. Although this serial test suite runs 30 minute quicker than the existing parallel test suite.